src/content/platform-integration/ios/app-extensions.md

```diff
@@ -22,6 +22,14 @@ For more information, check out
 [Issue #9690]:   {{site.github}}/flutter/website/issues/9690
 [Issue #135056]: {{site.github}}/flutter/flutter/issues/135056
 
+The iOS 18 release, coming in September 2024 and currently in beta,
```

Comment:
Consider removing time-specific language. This runs counter to [Google Style](https://developers.google.com/style/timeless-documentation).

```suggestion
The iOS 18 release
```